gym/f110_gym/envs/base_classes.py

```diff
@@ -360,10 +360,17 @@ def __init__(self, params, num_agents, seed, time_step=0.01, ego_idx=0):
         self.collisions = np.zeros((self.num_agents, ))
         self.collision_idx = -1 * np.ones((self.num_agents, ))
 
+        # obs info
+        self.obs_locations = None
+        self.obs_size = None
+
         # initializing agents
+        fov = np.pi 
+        n_beams = 540
```

Comment:
The default FoV and n_beams are based on the actual hardware. Is there a reason to use different numbers?

Reply:
Personally, I used different (smaller) numbers because I am training neural networks with the laser scan readings and I am using much fewer beams. 
I guess this probably doesn't have to be a commit on the remote branch, but I think it is useful should other people also want the speed increase and only need fewer beams.